YanZhylavy/HW7/Practical task 7.2.py

```diff
@@ -0,0 +1,43 @@
+def rectanle_area(len:float, wid:float)->float:
+  """
+  Function calculates area of a rectangle
+  using the lenght and width entered by the user 
+  """  
+  return len*wid
+
+def triangle_area(base:float, height:float)->float:
+  """
+  Function calculates area of a triangle
+  using the base and height entered by the user
+  """   
+  return 0.5*base*height
+
+def circle_area(rad:float)->float:
+  """
+  Function calculates area of a circle
+  using the radius entered by the user 
+  """
+  PI = 3.14  
+  return PI*rad**2
+    
+
+greetings = input("""Hello! With this programm you can
```

Comment:
greetings = input("""Hello! With this programm you can
calculate area of rectangle, triangle or circle. Please,select
a figure: """).lower()